backends/qualcomm/utils/utils.py

```diff
@@ -770,6 +770,7 @@ def generate_qnn_executorch_compiler_spec(
     online_prepare: bool = False,
     dump_intermediate_outputs: bool = False,
     profile: bool = False,
+    optrace: bool = False,
```

Comment:
How about change `profile: bool` to an enum class or something similar? Then users can choose profile level they want.

Reply:
Good point. However, before this change kProfileDetailed is the only supported profile level. It might be better to make this change in a separate diff.